goci-interfaces/goci-ui/src/main/resources/static/js/solrsearch.js

```diff
@@ -339,6 +339,9 @@ function processData(data) {
                     var variantDescription = '' // initializing empty description
 
                     var descriptionElements = descriptionTruncated.split("|");
+                    if (! descriptionElements[0]){
+                        descriptionElements[0] = "No description available.";
+                    }
```

Comment:
In the future, will this default text be added to the Solr document directly?

Reply:
Yes.